src/server/server.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+require('babel-core/register');
+const Hapi=require('hapi');
+// Create a server with a host and port
+const server=Hapi.server({
+    host:'localhost',
+    port:8000
+});
+
+server.route({
+    method: 'GET',
+    path: '/login/{username}',
+    handler: (request, h) => {
+        let status=validateuser(request.params.username);
+        console.log(status);
+        return h.response().code(status)
+    }
+});
+server.route({
+    method: 'GET',
+    path: '/Profile',
+    handler: (request, h) => {
+        return h.response({name: "swathi"}).code(200)
+
+    }
+});
+
+const validateuser = function (username) {
+    console.log(username);
+    if(username == 'swathi') {
```

Comment:
You should use === instead of ==. Also, consider using userName.toLowerCase() if this does not need to be case sensitive.